lib/netbox_client_ruby/api/dcim/inventory_item.rb

```diff
@@ -0,0 +1,15 @@
+require 'netbox_client_ruby/entity'
+require 'netbox_client_ruby/api/dcim/device'
+
+module NetboxClientRuby
+  class InventoryItem
+    include NetboxClientRuby::Entity
+
+    id id: :id
+    deletable true
```

Comment:
Are all of these coming from the entity class?

So many abstraction layers...Do we also need all of them?